.github/workflows/ci.yml

```diff
@@ -9,17 +10,15 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - name: Fetch the src
```

Comment:
Why did you add indentation instead of removing it? Why didn't you include a yamllint config as a part of the integration?

Reply:
Ok, I will fix it. So, should I add  `yamllint` to `ci.yml` too?